project/LibDependencies.scala

```diff
@@ -53,7 +52,6 @@ object LibDependencies {
   ) ++ shared
 
   private val itServerShared = Seq(
-    filters,
```

Comment:
this might have been causing some issues too I think, because we were importing play - not totally sure what the minimum required changes were to get intellij to load the build.sbt, we play-filters-helpers above instead

https://github.com/playframework/playframework/blob/a4615d8c2d020dfd38ee5e1c9a467e5357d8c4b2/dev-mode/sbt-plugin/src/main/scala/play/sbt/PlayImport.scala#L59